Formula/vsftpd.rb

```diff
@@ -44,11 +44,45 @@ def caveats
       s += <<-EOS.undent
         vsftpd was compiled with SSL support. To use it you must generate a SSL
         certificate and set 'enable_ssl=YES' in your config file.
+
       EOS
     end
 
+    s += <<-EOS.undent
+      vsftpd requires root privileges in most cases so you will need to run
```

Comment:
Why does it need root privileges in most cases? I'd rather we figured out a way to configure this and the `plist` such that it can be used without root privileges and those who want to use it as `root` can do so themselves.